docs/content/integrations/dbt/quickstart.mdx

```diff
@@ -0,0 +1,264 @@
+---
+title: "Quickstart: Run your dbt project with Dagster"
+description: Get started swiftly with this simple dbt + Dagster example.
+---
+
+# Quickstart: Run your dbt project with Dagster
+
+<Note>
+  <strong>Note</strong>: This guide uses the <code>DbtProject</code> class,
+  which is an experimental feature. Visit the{" "}
+  <a href="/\_apidocs/libraries/dagster-dbt">
+    dagster-dbt library API reference
+  </a>{" "}
+  for more info.
+</Note>
+
+If you just want to see the code of the example, skip to the **Create a single Dagster file** section.
```

Comment:
This sentence can probably can be omitted, given that it's the default option in "Load your dbt project"